Sources/App/Controllers/API/APISearchController.swift

```diff
@@ -14,13 +14,23 @@ extension API {
 
 
 extension API {
+    static let searchLimit = 20
+    static let searchLimitLeeway = 5
```

Comment:
The idea here is to allow a lilttle leeway so we don't cut off at 20 when there are 21 results. We'll return the full result set up to 24 and from 25+ we'll trim back to 20 and set `hasMoreResults` to `true.

Happy to cut off straight at 20 as well if you'd rather have a fixed value.

Reply:
No, I think this is smart. 👍